core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -168,6 +236,50 @@ protected void doStop() throws Exception {
     return server;
   }
 
+  private static String[] getList(String original) {
+    String[] parts = original.split(",");
+    for (int i = 0; i < parts.length; i++) {
+      parts[i] = parts[i].trim();
+    }
+    return parts;
+  }
+
+  // TODO: delete deprecatedPort parameter when `PORT_CONFIG` is deprecated. It's only used to support the deprecated
+  //       configuration.
+  static List<URI> parseListeners(String listenersStr, int deprecatedPort) {
+    String[] parts = getList(listenersStr);
+
+    // handle deprecated case, using PORT_CONFIG.
+    // TODO: remove this when `PORT_CONFIG` is deprecated, because LISTENER_CONFIG will have a default value which
+    //       includes the default port.
+    if (parts.length == 0 || parts[0].isEmpty()) {
+      parts = new String[] {"http://0.0.0.0:" + deprecatedPort};
```

Comment:
Do you need `0.0.0.0`? I suppose it is more explicit, although I don't really know if the preferred approach is to bind all interfaces, bind the FQDN, or maybe something else. Do you remember what Kafka does here? I think it doesn't even include an explicit IP, although maybe that's just a convenience it provides in its configs.